common/common.scss

```diff
@@ -37,3 +37,7 @@
     }
   }
 }
+
+li.sidebar-section-link-wrapper[data-list-item-name="New Topic"] {
```

Comment:
that won't work with translated content right?

Reply:
Yes you are right. I think for now it's fine for dev.